core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -421,10 +424,32 @@ public void start(boolean isLeader) throws IOException {
       mLostFilesDetectionService = getExecutorService().submit(new HeartbeatThread(
           HeartbeatContext.MASTER_LOST_FILES_DETECTION, new LostFilesDetectionHeartbeatExecutor(),
           Configuration.getInt(PropertyKey.MASTER_HEARTBEAT_INTERVAL_MS)));
+      mStartupConsistencyCheck = getExecutorService().submit(new Callable<List<AlluxioURI>>() {
```

Comment:
ahhh, I see. I thought it was in the constructor but I was mistaken. This all makes sense now.